src/install/apache/whatsNext.mdx

```diff
@@ -0,0 +1,637 @@
+---
+componentType: default
+headingText: Metrics collected by the integration
+---
+
+### Apache configuration options [#config-options]
+
+The Apache integration collects both metrics and inventory information. This table provides a description for each config setting and whether it applies to metrics, inventory, or both.
+
+<table>
```

Comment:
I'd chunk the "applies to" like with like. e.g., lead with 'metrics, inventory' at the top, then all the inventory, then all the metrics. 

The intrro section seems to bucket these configs by metrics/inventory/both, but the table lists the 'applies to' last. i wonder if we should have the first column be 'applies to'. I'll leave it up to you tho! just a thought

Reply:
I think the column should stay at the end since folks tend to skim the through the left-most column to find things, but I chunked all the related metrics/inventory ones together!